Release_Notes.md

```diff
@@ -14,11 +14,45 @@ See the License for the specific language governing permissions and<BR>
 limitations under the License.*_<BR>
 
 # **Release Notes** #
-_*Last Update: 2024/01/18*_
+_*Last Update: 2024/04/08*_
 
-## Rev 1p0 ##
+## Rev 1p1 ##
 
-### Rev 1p0 release date: 2024/01/18 ###
+#### Rev 1p1 release date: 2024/04/10 ####
+- Caliptra Hardware Specification: Updated with LMS accelerator, ECC radix changes
+- Caliptra Integration specification updates to add clarity
+- Caliptra README updates to tool versions, simulation instructions
+- Update RDL register descriptions for clarity, accuracy in generated reg document
+- Verification
+  - Update zeroize assertions
+  - Add X check assertions to all register files
+  - New assertions and firmware tests for LMS accelerator
+  - Formal: ???
```

Comment:
incomplete update here?

Reply:
Yes, I made the PR a draft, I need to flesh out a few of these lines.

Reply:
Added more details here